libs/ui/src/components/molecules/FieldWrapper/FieldWrapper.tsx

```diff
@@ -44,7 +44,7 @@ export const FieldWrapper = ({
   return (
     <FieldWrapperBase className={classes}>
       <InputLabel required={required} info={info} id={id}>
-        {label}
+        {label ? label : 'No label found'}
```

Comment:
This was causing an error when I pulled down Develop. In the type definition label is optional causing the build to fail.